### PR TITLE
Pagination phase 2

### PR DESCRIPTION
**Pull request by Rosa-Fox**
Paginates content-items by utilising the pagination functionality from the publishing-api. Uses the Kaminari gem for pagination related styling.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,10 @@ gem 'mongoid', '5.0.1'
 gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails', '~> 3.1.4'
 gem 'select2-rails', '~> 4.0.0'
+gem 'sidekiq', '3.2.1'
```
Comment (christophwong) [inline Gemfile]: I don't think we are using sidekiq anywhere here.

**Comment by christophwong** [comment]
Basic pagination functionality works well. However, when I tried using `per_page = 10` I only saw 4 documents (same with other numbers). It may be some sort of filtering at publishing-api level. 
cc @dougdroper

**Comment by dougdroper** [comment]
@christophwong sure, let's look at this together.

**Comment by Rosa-Fox** [comment]
Hi,

Looked at this with @dougdroper on Friday to see if there could have been anything implemented on publishing-api that could have caused extra filtering on some finders. When testing on Doug's environment he was receiving the correct number of paginated content items. We deployed the branch to integration to try and figure out if the mis-match in number of results on @christophwong's local environment was due to a data mismatch and it worked fine on there so I think all is ok.

I have only implemented pagination for finders, not manuals. I will do that now on a separate branch and PR.

**Comment by issyl0** [comment]
This seems good, now I've understood why we're doing `page.to_i.to_s`. 👍

**Pull request merged**